backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxBackend.scala

```diff
@@ -140,22 +140,22 @@ object BackendSettings extends BackendSettingsApi {
       format: FileFormat,
       fields: Array[StructField],
       bucketSpec: Option[BucketSpec],
-      options: Map[String, String]): Option[String] = {
+      options: Map[String, String]): ValidationResult = {
 
     def validateCompressionCodec(): Option[String] = {
       // Velox doesn't support brotli and lzo.
       val unSupportedCompressions = Set("brotli, lzo")
       val compressionCodec = WriteFilesExecTransformer.getCompressionCodec(options)
       if (unSupportedCompressions.contains(compressionCodec)) {
-        Some("brotli or lzo compression codec is not support in velox backend.")
+        Some("brotli or lzo compression codec is not support in Velox backend.")
```

Comment:
`brotli`  -> `Brotli`   and `support`  -> `supported`

Reply:
ok.